brainvisa/toolboxes/data management/processes/import/volBrain/import_vol2brain.py

```diff
@@ -0,0 +1,252 @@
+                            'space': 'native'},
+        section=outputs_native),
+    'native_macrostructures', WriteDiskItem(
+        'Split Brain Mask',
+        'gz compressed NIFTI-1 image',
+        requiredAttributes={'modality': 'vol2Brain',
+                            'space': 'native'},
+        section=outputs_native),
+    'native_tissues', WriteDiskItem(
+        'Intracranial labels',
+        'gz compressed NIFTI-1 image',
+        requiredAttributes={'modality': 'vol2Brain',
+                            'space': 'native'},
+        section=outputs_native),
+    'native_mask', WriteDiskItem(
+        'Intracranial mask',
+        'gz compressed NIFTI-1 image',
+        requiredAttributes={'modality': 'vol2Brain',
+                            'space': 'native'},
+        section=outputs_native),
+    'native_lobes', WriteDiskItem(
+        'Brain Lobes',
+        'gz compressed NIFTI-1 image',
+        requiredAttributes={'modality': 'vol2Brain',
+                            'space': 'native'},
+        section=outputs_native),
+    'native_thickness', WriteDiskItem(
+        'Cortical Thickness map',
+        'gz compressed NIFTI-1 image',
+        requiredAttributes={'modality': 'vol2Brain',
+                            'space': 'native'},
+        section=outputs_native),
+    'native_t1', WriteDiskItem(
+        'T1 MRI Denoised and Bias Corrected',
+        'gz compressed NIFTI-1 image',
+        requiredAttributes={'modality': 'vol2Brain',
+                            'space': 'native'},
+        section=outputs_native),
+)
+
+
+def initialization(self):
+    
+    def linkVolBrainOutput(self, proc):
+        if self.subject and self.acquisition:
+            d = {'_database': self.subject.get('_database'),
+                 '_ontology': 'brainvisa-3.2.0',
+                 'center': self.subject.get('center'),
+                 'subject': self.subject.get('subject'),
+                 'acquisition': self.acquisition}
+            return self.signature['report_csv'].findValue(d)
+    
+    def linkLobesMNI(self, proc):
+        if self.mni_structures:
+            p = self.mni_structures.fullPath().replace('structures',
+                                                       'lobes')
+            return p
+    
+    def linkLobesNative(self, proc):
+        if self.native_structures:
+            p = self.native_structures.fullPath().replace('structures',
+                                                          'lobes')
+            return p
+    
+    def linkThicknessMNI(self, proc):
+        if self.mni_structures:
+            p = self.mni_structures.fullPath().replace('structures',
+                                                       'thickness')
+            return p
+    
+    def linkThicknessNative(self, proc):
+        if self.native_structures:
+            p = self.native_structures.fullPath().replace('structures',
+                                                          'thickness')
+            return p
```

Comment:
Since you create a type for those, we don't need those link functions.

Reply:
You're right, I did not check all the process when I created the new type